Server/API/login/postLogin.go

```diff
@@ -0,0 +1,41 @@
+package login
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/labstack/echo/v4"
+	entities "ishtaloo.io/Entities"
+)
+
+func postLogin(c echo.Context) (err error) {
+	cookie := new(http.Cookie)
+	cookie.Name = "userId"
+	u := new(entities.LoginRequest)
+	if err = c.Bind(u); err != nil {
+		return err
+	}
+	userName := entities.LoginRequest{
+		UserName: u.UserName,
+	}
+	cookie.Value, _ = createToken(userName)
```

Comment:
_ is for error?

Reply:
woops I wanted to reference that error later and I forgot
I will add it now

Reply:
changed it in a newer commit